jfrog_platform_self_hosted/README.md

```diff
@@ -0,0 +1,75 @@
+<div class="alert alert-warning">The existing agent check to gather JFrog metrics has been replaced with Fluentd. The agent check will be deprecated on August 31st 2023.</div>
```

Comment:
@MahithaB This line needs to be updated to the new date that Shailendra agreed on.